content/projects/java-specific/functional-interface/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - topic/oop
```

Comment:
please make sure that these paths are correct. They need to be exactly right.

There isn't a directory called `topic`. And even if I look inside the `topics` directory then there isn't anything named `oop`. Maybe you mean `projects/oop` but that project has type nosubmit. 

So maybe you mean each of the projects inside the oop directory... Can you list out exactly what is required?